mmdet/models/layers/transformer/dab_detr_transformer.py

```diff
@@ -0,0 +1,300 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
considering renaming the following files:
dab_detr_transformer -> dab_detr_layers.py
detr_transformer -> detr_layers.py
deformable_detr_transformer -> deformable_detr_layers.py